ui/imports/shared/views/chat/ChatTextView.qml

```diff
@@ -24,7 +24,21 @@ Item {
 
     z: 51
 
-    implicitHeight: visible ? (showMoreLoader.active ? childrenRect.height - 10 : chatText.height) : 0
+    implicitHeight: {
+        if (visible) {
```

Comment:
I wonder if using form like
```
if(!visible) return 0
...
```
is better ?

Reply:
@alexandraB99 Not sure if very good reason. But I remember some practices about adding in the beginning the conditions which lead to exit from function and another one was about nesting levels, which you should keep lower.

Reply:
updated!